glottolog3/adapters.py

```diff
@@ -39,15 +39,54 @@ def rec(self, ctx, req):
 class LanguoidCsvDump(CsvDump):
 
     def query(self, req):
-        return self.model.csv_query(session=req.db)
+        import sqlalchemy as sa
```

Comment:
Can we avoid local imports? Makes the code a bit more difficult to inspect by pycharm or grep.

Reply:
Thanks. Pushed a new version that also omits the `newick` and `hid` columns.